components/ws-proxy/pkg/proxy/routes.go

```diff
@@ -393,22 +356,19 @@ func staticTheiaResolver(config *Config, req *http.Request) (url *url.URL, err e
 	return &targetURL, nil
 }
 
-func dynamicTheiaResolver(infoProvider WorkspaceInfoProvider) targetResolver {
-	return func(config *Config, req *http.Request) (res *url.URL, err error) {
-		coords := getWorkspaceCoords(req)
-		info := infoProvider.WorkspaceInfo(coords.ID)
-		if info == nil {
-			log.WithFields(log.OWI("", coords.ID, "")).Warn("no workspace info available - cannot resolve Theia route")
-			return nil, xerrors.Errorf("no workspace information available - cannot resolve Theia route")
-		}
+func dynamicTheiaResolver(config *Config, req *http.Request) (res *url.URL, err error) {
```

Comment:
`theia` still pops up here and there in function names and comments which are relevant to IDEs in general, not sure whether it was an intention of the PR to clean it up too

Reply:
it wasn't, but maybe it should :)

Reply:
it still isn't all gone, but it's better now